dipy/workflows/viz.py

```diff
@@ -93,8 +93,8 @@ def change_slice(obj, event):
         slider = widget.slider(show_m.iren, show_m.ren,
                                callback=change_slice,
                                min_value=0,
-                               max_value=image_actor.shape[1] - 1,
-                               value=image_actor.shape[1] / 2,
+                               max_value=image_actor.shape[2] - 1,
```

Comment:
Should use `shape[2]` here since previously (line 91) we did `image_actor.display(None, None, z)`.